Doc/conf.py

```diff
@@ -454,6 +454,25 @@
 # https://github.com/sphinx-doc/sphinx/issues/12359
 epub_use_index = False
 
+# translation tag
+# ---------------
+
+language_code = None
+for arg in sys.argv:
+    if arg.startswith('language='):
+        language_code = arg.split('=', 1)[1]
+
+if language_code:
+    tags.add('translation')  # noqa: F821
+
+    rst_epilog += f"""\
+.. _TRANSLATION_REPO: https://github.com/python/python-docs-{language_code}
```

Comment:
Beware, language_code here is in iso639 format, like: `-D language=bn_IN`.

But the repo URL uses a lowercased-with-dash format, like https://github.com/python/python-docs-bn-in.

According to [docsbuild_script](https://github.com/python/docsbuild-scripts/blob/7787668fb2deaa8ea6febfb0f45667fcc489f04d/build_docs.py#L295) the conversion is as simple as:

```python
    @property
    def tag(self) -> str:
        return self.iso639_tag.replace("_", "-").lower()
```